roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
Does `openshift_inventory` get called multiple times during a run of the install playbook?